jbpm-test/src/main/java/org/jbpm/test/JbpmJUnitBaseTestCase.java

```diff
@@ -933,6 +936,10 @@ public void addEnvironmentEntry(String name, Object value) {
     	customEnvironmentEntries.put(name, value);
     }
 
+    public void addPersistenceProperty(String name, Object value) {
```

Comment:
I was just following the name of the method above which also puts entries into Map and starts with  `add`. But I agree that `set` makes more sense in this case.